src/Mapbender/WmsBundle/Element/WmsLoader.php

```diff
@@ -194,4 +200,19 @@ protected function getWmsSource($request)
 
         return $importerResponse->getWmsSourceEntity();
     }
+
+    protected function splitLayers($layerConfiguration)
+    {
+        $children = $layerConfiguration['configuration']['children'][0]['children'];
+        foreach ($children as $child) {
+            $layerConfiguration['configuration']['children'][0]['children'] = [$child];
+            $layerConfiguration['configuration']['children'][0]['options']['title'] = $child['options']['title']
+                . ' ('
+                . $layerConfiguration['configuration']['title']
+                . ')'
+            ;
+            $layerConfigurations[] = $layerConfiguration;
```

Comment:
That's a notice. Please preinitialize `$layerConfigurations = array();` before the loop